common/app/views/fragments/amp/stylesheets/main.scala.html

```diff
@@ -282,3 +282,7 @@
     @Html(common.Assets.css.amp)
 
 </style>
+
+@if(play.Play.isDev() || !InlineCriticalCss.isSwitchedOn) {
```

Comment:
Actually this isn't our main mechanism for inlining css in AMP,  that is above. I think this needs a comment to explain that.  

This also won't pass validation in dev because this relies on a link tag which isn't allowed in amp. 

I am torn whether this is actually a good idea. It really does improve the developer experience, but does actually add complexity when checking if the amp pages are valid.